omero/users/clients-overview.txt

```diff
@@ -16,17 +16,18 @@ installations worldwide, OMERO has been shown to be relatively easy to install
 and get running.
 
 OMERO.insight and OMERO.importer are desktop applications written in Java
-and require Java 1.7 (or higher) to be installed on the user's computer
-(automatically installed on most up-to-date OS X and Windows systems).
+and require Java |javaversion| (or higher) to be installed on the user's
+computer (automatically installed on most up-to-date OS X and Windows
+systems).
```

Comment:
It's not installed by default on Windows (and OS X has an old java 1.6 for some reason)